api/specs.proto

```diff
@@ -304,10 +304,8 @@ message EndpointSpec {
 	repeated PortConfig ports = 2;
 }
 
-// NetworkSpec specifies user defined network parameters.
-message NetworkSpec {
-	Annotations annotations = 1 [(gogoproto.nullable) = false];
-
+// CNMNetworkSpec specifies user defined network parameters for CNM networks
+message CNMNetworkSpec {
 	// DriverConfig specific configuration consumed by the network driver.
 	Driver driver_config = 2;
```

Comment:
You'll still have to support all of these fields on `NetworkSpec` to be backwards compatible.